frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -460,3 +470,65 @@ action.GitMachete.CheckoutParentAction.undefined.branch-name=Branch (''{0}'') do
 action.GitMachete.CheckoutFirstChildAction.text=Git Machete: Checkout First Child Branch
 action.GitMachete.CheckoutFirstChildAction.description=Checkout first child branch (''{0}'')
 action.GitMachete.CheckoutFirstChildAction.undefined.branch-name=Branch (''{0}'') does not have any children in the branch layout
+
+# Traverse
+
+action.GitMachete.BaseTraverseAction.description=Traverse
+action.GitMachete.BaseTraverseAction.description.empty-layout=Cannot traverse an empty branch layout
+
+action.GitMachete.BaseTraverseAction.dialog.traverse-approval.title=Traverse Info
+action.GitMachete.BaseTraverseAction.dialog.traverse-approval.text.HTML=<html>\
+  This action triggers an interactive walk over the branches and suggests actions (pushes/pulls/rebases) \
```

Comment:
Theoretically, slide-out could also be mentioned here... but maybe it'd be too verbose, esp. for the first-time viewers